eidas-saml-parser/src/main/java/uk/gov/ida/notification/eidassaml/EidasSamlParserConfiguration.java

```diff
@@ -19,19 +19,28 @@
 
     @JsonProperty
     @Valid
-    private ReplayCheckerConfiguration replayChecker = new ReplayCheckerConfiguration();
+    private final ReplayCheckerConfiguration replayChecker = new ReplayCheckerConfiguration();
 
+    @Valid
+    @NotNull
     @JsonProperty
+    private final JerseyClientConfiguration clientConfig = new JerseyClientConfiguration();
```

Comment:
I think that the `clientConfig` field can be dropped, as well as the getter for it below. The only place it would be used is in the construction of the Metatron proxy, and we don't use it there.

Reply:
Alternatively I guess we should pull this configuration in, so that's it's customisable via the config but has a default?

Reply:
I've gone with taking it out.  If it turns out we do need it, we can put it back in but less is more where possible.